src/dictionary.c

```diff
@@ -38,13 +38,11 @@ int Dictionary_Add(RedisModuleCtx *ctx, const char *dictName, RedisModuleString
   return valuesAdded;
 }
 
-int Dictionary_Del(RedisModuleCtx *ctx, const char *dictName, RedisModuleString **values, int len,
-                   char **err) {
```

Comment:
No. We don't use it. It was removed in https://github.com/RediSearch/RediSearch/pull/5231
Now, in case we can't open the dictionary, we don't return an error, we return 0.